smoketests/pages/android_feedback_form.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from pages.feedback import FeedbackPage
+
+
+class AndroidFeedbackFormPage(FeedbackPage):
+    _page_title = 'Firefox Feedback :: Firefox Input'
+
+    def go_to_feedback_page(self,
+                            version="",
+                            channel="",
+                            last_visited="",
+                            in_device=False):
+        url = self.base_url + '/feedback/android'
+        params = []
+
+        if version:
+            url = '%s/%s' % (url, version)
+
+        if channel:
+            url = '%s/%s' % (url, channel)
+
+        if in_device:
+            params.append('utm_source=feedback-prompt')
+
+        if last_visited:
+            params.append('url=%s' % (last_visited))
+
+        if len(params) > 0:
+            url = '%s/?%s' % (url, '&'.join(params))
+
+        print('Requesting url: %s' % url)
+
+        self.selenium.get(url)
+        self.is_the_current_page
+
+    # Intro card
+    _happy_button_locator = (By.CSS_SELECTOR, '#intro #happy-button')
+    _sad_button_locator = (By.CSS_SELECTOR, '#intro #sad-button')
+
+    def click_happy_feedback(self):
+        self.selenium.find_element(*self._happy_button_locator).click()
+
+    # Happy thanks
+    _playstore_locator = (By.CSS_SELECTOR, '#play-store')
+
+    @property
+    def playstore_link_is_http_release(self):
+        href = self.selenium.find_element(
+            *self._playstore_locator
+        ).get_attribute('href')
+        return href == 'https://play.google.com/store/apps/details?id=org.mozilla.firefox#details-reviews'
```

Comment:
I think this should instead return the href, and the test itself should contain the URL you're asserting.